AlertaDengue/locale/en/LC_MESSAGES/django.po

```diff
@@ -1152,6 +1123,173 @@ msgstr "Green Level"
 msgid "Casos Notificados de %(disease_label)s"
 msgstr "Reported cases of %(disease_label)s"
 
+#: dados/templates/services.html:19
+msgid "Serviços de Dados Públicos"
+msgstr "Public Data Service"
+
+#: dados/templates/services_api.html:69
+#: dados/templates/services_api_doc.html:46
+#: dados/templates/services_maps.html:69
+#: dados/templates/services_maps_doc.html:47 templates/base.html:74
+msgid "Dados"
+msgstr "Data"
+
+#: dados/templates/services_api.html:69
+msgid "API"
+msgstr ""
+
+#: dados/templates/services_api.html:79
+msgid "Para mais informações, acesse a documentação do serviço da API."
+msgstr "For more information, access the map service documentation"
```

Comment:
should be:

"For more information, access the API service documentation."

Reply:
yes

Reply:
ok